test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -22,30 +23,97 @@
+    # get rowan contract address
+    rowan_token_address = burn_rowan_get_erc20_address(ctx)
+
+    print("+++++++++++++++++++++++++ ", rowan_token_address)
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
+
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, rowan_token_address, isRowan=True)
+
+# test single sif account burn erc20 to multiple ethereum accounts
+def test_single_sif_to_multiple_eth_account_burn_erc20(ctx: test_utils.EnvCtx):
+    # deploy an erc20 contract
+    token_decimals = 18
+    total_amount = 10 ** 28
+    token_data: test_utils.ERC20TokenData = ctx.generate_random_erc20_token_data()
+    erc20_sc = ctx.deploy_new_generic_erc20_token(token_data.name, token_data.symbol, token_decimals)
+    # mint token to operator account
+    ctx.mint_generic_erc20_token(erc20_sc.address, total_amount, ctx.operator) 
+    ctx.advance_blocks()
+
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
 
-def test_load_tx_ethbridge_burn_short(ctx: test_utils.EnvCtx):
-    _test_load_tx_ethbridge_burn(ctx, 123456 * eth.GWEI, [[2, 2], [2, 2]])
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, erc20_sc.address)
 
+# test single sif account burn ceth to multiple ethereum accounts
+def test_single_sif_to_multiple_eth_account_burn_eth(ctx: test_utils.EnvCtx):
+    transfer_table = build_transfer_table()
+    amount_per_tx = 1000100101
 
-def test_load_tx_ethbridge_burn(ctx: test_utils.EnvCtx):
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+# short test to verify eth burn works
+def test_load_tx_ethbridge_burn_eth_short(ctx: test_utils.EnvCtx):
+    transfer_table = [[2, 2], [2, 2]]
+    amount_per_tx = 1000100101
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+# test multiple sif accounts burn ceth to multiple ethereum accounts
+def test_load_tx_ethbridge_burn_eth(ctx: test_utils.EnvCtx):   
     # Matrix of transactions that we want to send. A row (list) in the table corresponds to a sif account sending
     # transactions to eth accounts. The numbers are transaction counts, where each transaction is for amount_per_tx.
     # Each sif account uses a dedicated send thread.
     transfer_table = [
-        [100, 100, 100],
-        [100, 100, 100],
-        [100, 100, 100],
-        [10, 20, 30],
-    ]
-
-    amount_per_tx = 123456 * eth.GWEI
-
-    _test_load_tx_ethbridge_burn(ctx, amount_per_tx, transfer_table)
+            [100, 100, 100],
+            [100, 100, 100],
+            [100, 100, 100],
+            [10, 20, 30],
+        ]
+    amount_per_tx = 1000100101
+    _test_load_tx_ethbridge_lock_burn(ctx, amount_per_tx, transfer_table, None)
+
+def _test_load_tx_ethbridge_lock_burn(ctx: test_utils.EnvCtx, amount_per_tx: int, 
+    transfer_table: List[List[int]], token_address: Address, isRowan: bool = False, randomize: bool = None):
```

Comment:
`token_address: Optional[Address]` is better since it can be null.
Or (depending on style preferences) also `Union[Address, None]`